### DISCUSSION TITLE
Allow Dependabot to Open Issues In Lieu of Pull Requests

### DISCUSSION THREAD

**Discussion by nategoethel** [community/community#75433] [Code Security]
### Select Topic Area

Product Feedback

### Body

It would be nice to be able to configure Dependabot to open issues instead of pull requests. Our organization has a fairly complicated dependency structure which requires re-compiling our dependency lists when a package is changed to address a Dependabot alert. However, Dependabot cannot be configured to run these recompilation steps, so we would prefer to just auto-generate issues and let our teams handle the package changes.

In the meantime, we'll be using a Github Action or Lambda function to capture the Dependabot webhook events and create these issues ourselves, but we would love an official mechanism for this!

**Comment by carogalvin** [comment]
@nategoethel thanks for this feedback!
